arrow_pd_parser/export.py

```diff
@@ -67,3 +68,33 @@ def pd_to_json(
             new[col].replace("NaT", np.nan, regex=False, inplace=True)
 
     new.to_json(output_file, orient=orient, lines=lines, **kwargs)
+
+
+def pd_to_parquet(
+    df: pd.DataFrame,
+    output_file: Union[IO, str],
+    arrow_schema: pa.lib.Schema,
```

Comment:
Shouldn't be mandatory.

Also I would add `from_pandas_kwargs` and `write_table_kwargs` parameters which can be put into the respective function calls

Reply:
https://arrow.apache.org/docs/python/generated/pyarrow.parquet.write_table.html#pyarrow.parquet.write_table

Reply:
has been made non-mandatory